server/game/cards/01_SOR/events/YoureMyOnlyHope.ts

```diff
@@ -0,0 +1,40 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { CostAdjustType } from '../../../core/cost/CostAdjuster';
+
+export default class YoureMyOnlyHope extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3509161777',
+            internalName: 'youre-my-only-hope',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Look at the top card of your deck',
+            immediateEffect: AbilityHelper.immediateEffects.lookAt((context) => ({
+                target: context.source.controller.getTopCardOfDeck()
+            })),
+            then: (thenContext) => ({
+                title: thenContext.source.controller.base.remainingHp <= 5
+                    ? 'Play it for free'
+                    : 'Play it, it costs 5 less',
```

Comment:
Might be good to insert the name of the card into the title to help the player remember the card they saw